lms/envs/aws.py

```diff
@@ -917,6 +917,11 @@
     ENTERPRISE_ENROLLMENT_API_URL
 )
 
+# Enterprise logo image size limit in KB's
+ENTERPRISE_CUSTOMER_LOGO_IMAGE_SIZE = ENV_TOKENS.get(
+    'ENTERPRISE_CUSTOMER_LOGO_IMAGE_SIZE'
+)
```

Comment:
I agree. We should set a base value in `common.py`, and then use that as a default for the variable here.